libvirt/tests/src/virsh_cmd/pool/virsh_find_storage_pool_sources.py

```
@@ -60,6 +61,30 @@ def run(test, params, env):
+            polkit.setup()
+            logging.info("Polkit setup completed successfully")
+            
+            # Explicitly restart virtstoraged for storage driver ACL tests
+            # The libvirtd.restart() in polkit.setup() doesn't always restart virtstoraged
+            from virttest.staging import service
+            try:
+                virtstoraged = service.Factory.create_service("virtstoraged")
+                logging.info("Restarting virtstoraged to apply polkit rules")
+                virtstoraged.restart()
+                logging.info("virtstoraged restarted successfully")
+            except Exception as svc_err:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)